### PR TITLE
Updates privacy policy

### PR DESCRIPTION
**Pull request by jsiems**
Check for-
spelling errors
stupid things I say
things that don't make sense
things you expect to be in there but aren't

Be critical please!

### CODE REVIEW

File: templates/questions/index.html.ep [hunk 1]
```diff
@@ -291,7 +291,10 @@ $(document).ready(function() {
 			Privacy Policy
 		</div>
 		<div class="privacyContent">
-			Here is the content of the privacy policy. It is kinda long, so we gotta make sure it wraps. Hopefully people really like this privacy policy. Hopefully it does not scare people away.
+			AMA uses "cookies" to enhance User experience. User's web browser places cookies on their hard drive for record-keeping purposes. Users are still completely anonymous. User may choose to set their web browser to refuse cookies, or to alert when cookies are being sent. If cookeis are disabled, note that some parts of the AMA site may not function properly.
```
Comment (s1037989) [inline templates/questions/index.html.ep]: Explain "record-keeping purposes"?

Explain how users are anonymous when cookies are used?

cookeis should be spelled cookies.

If anonymity is a feature, why are cookies necessary?  Can you make the system behave the same without cookies?  What functionality is lost if cookies are disabled?

File: templates/questions/index.html.ep [hunk 2]
```diff
@@ -291,7 +291,10 @@ $(document).ready(function() {
...
+			<br><br>AMA also gives the user the option to sign in. When the user is signed in, the User is still entirely anonymous to all other users. However, when signed, it is possible for site moderators to track down specific people thorugh the database if neccesarry. Notice that if a user signs in, he/she loses an element of anonymity.
```
Comment (s1037989) [inline templates/questions/index.html.ep]: Per our excellent debate the other day, I thought users do _not_ sign in to AMA?  But it seems your description here claims they do?

> it is possible for site moderators to track down specific people thorugh the database if neccesarry

thorugh -> through
neccesarry -> necessary

Why would it ever be "_necessary_"??

**Comment by s1037989** [comment]
This is remarkable!  It really communicates all of the necessary elements in a very straightforward and transparent way.

:+1:

**Comment by jsiems** [comment]
Update 2.0, what do you guys think? I think I addressed everything.

**Comment by andygoodday** [comment]
👍

**Comment by s1037989** [comment]
:+1: Fantastic!!  Love it!!

**Pull request merged**